Fix Cloud documentation structure and consolidate issue resolver information (#8558)
All-Hands-AI/OpenHands: docs/modules/usage/cloud/github-installation.md

# GitHub Installation

This guide walks you through the process of installing and configuring OpenHands Cloud for your GitHub repositories.

## Prerequisites

- A GitHub account
- Access to OpenHands Cloud

## Installation Steps

1. Log in to [OpenHands Cloud](https://app.all-hands.dev)

> 2. If you haven't connected your GitHub account yet:
>    - Click on `Connect to GitHub`
>    - Review and accept the terms of service
>    - Authorize the OpenHands AI application

Reviewer: ```suggestion
2. If you haven't connected your GitHub account yet:
   - Click on `Log in with GitHub`
   - Authorize the OpenHands AI application
```

Author: I think the original is correct?
<img width="481" alt="Screenshot 2025-05-20 at 10 14 14 PM" src="https://github.com/user-attachments/assets/c436c247-ef5f-41f1-b4f0-f39c454d39da" />

Reviewer: Huh. Weird! I'm not getting the terms of service anymore. Is it because I have already accepted it?

## Adding Repository Access

You can grant OpenHands access to specific repositories:

> 1. Click on the `Select a GitHub project` dropdown, then select `Add more repositories...`

Author: ```suggestion
1. Click on `Add GitHub repos`
```

2. Select your organization and choose the specific repositories to grant OpenHands access to.
   - OpenHands requests short-lived tokens (8-hour expiration) with these permissions:
     - Actions: Read and write
     - Administration: Read-only
     - Commit statuses: Read and write
     - Contents: Read and write
     - Issues: Read and write
     - Metadata: Read-only
     - Pull requests: Read and write
     - Webhooks: Read and write
     - Workflows: Read and write
   - Repository access for a user is granted based on:
     - Permission granted for the repository
     - User's GitHub permissions (owner/collaborator)
3. Click `Install & Authorize`

## Modifying Repository Access

> You can modify repository access at any time:
> * Using the same `Select a GitHub project > Add more repositories` workflow, or
> * By visiting the Settings page and selecting `Configure GitHub Repositories` in the `GitHub Settings` section.

Reviewer: ```suggestion
You can modify repository access at any time by visiting the Settings page and selecting `Configure GitHub Repositories` under the `Git` tab.
```
